modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -44,61 +44,65 @@ $ echo -ne "root" | base64
 $ echo -ne "password" | base64
 ----
 
-. Create a configuration file for the bare metal node.
-+
-[source,terminal]
-----
-$ vim bmh.yaml
-----
+. Create a configuration file for the bare metal node using the following example `bmh.yaml` file, replacing values in the YAML to match your environment:
 +
 [source,yaml]
 ----
 ---
+apiVersion: v1
```

Comment:
@ktothill I just had a look at this using rich diff, and I'm not sure if it's just the display but the alignment (spacing) is incorrect for the first secret. It should be similar to this:

type: Opaque
stringData:
nmstate: |
routes:
config:

destination: 0.0.0.0/0
next-hop-address: 192.168.123.1
next-hop-interface: enp0s4
dns-resolver:
config:
server:
- 192.168.123.1
interfaces:
name: enp0s4
state: up
ipv4:
address:
ip: 192.168.123.16
prefix-length: 24
enabled: true
dhcp: false
In addition, the last line (preprovisioningNetworkDataName) needs to be aligned with rootDeviceHints